python/tidy/servo_tidy/tidy.py

```diff
@@ -723,13 +725,32 @@ def parse_config(content):
     config['ignore']['directories'] = map(lambda path: os.path.join(*path.split('/')),
                                           config['ignore']['directories'])
 
+    # Add dict of dir, list of expected ext to config
+    dirs_for_ext = config_file.get("check_dirs_for_exts", {})
```

Comment:
Since we're using `get` with defaults, maybe we don't want it in the default config?